### PR TITLE
Clarifying that this project is community driven

### PR DESCRIPTION
**Pull request by johanandren**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ A replicated [Akka Persistence](http://doc.akka.io/docs/akka/2.4.0/scala/persist
 - This plugin implements both a journal as well as a snapshot store,
 - Please note, however, that it does not include an Akka Persistence Query plugin.
 
+This plugin is "Community Driven" https://developer.lightbend.com/docs/lightbend-platform/introduction/getting-help/support-terminology.html#community-driven
+
```
Comment (patriknw) [inline README.md:11]: There is already a section about Support at the end, but I guess it doesn't hurt to mention it again at the top:

> This project is *community maintained*. The Lightbend subscription does not cover support for this project.

File: README.md [hunk 2]
```diff
@@ -7,6 +7,8 @@ A replicated [Akka Persistence](http://doc.akka.io/docs/akka/2.4.0/scala/persist
 - This plugin implements both a journal as well as a snapshot store,
 - Please note, however, that it does not include an Akka Persistence Query plugin.
 
+This plugin is "Community Driven" https://developer.lightbend.com/docs/lightbend-platform/introduction/getting-help/support-terminology.html#community-driven
```
Comment (patriknw) [inline README.md]: should that be a proper link instead of showing the full url?

File: README.md (hunk 1 above)
Comment (johanandren) [reply README.md:11]: Ah, missed that somehow. I think that is enough then.

File: README.md (hunk 2 above)
Comment (raboof) [inline README.md]:
```suggestion
This plugin is [Community Driven](https://developer.lightbend.com/docs/lightbend-platform/introduction/getting-help/support-terminology.html#community-driven)
```

**Review by raboof — APPROVED** [review]

**Comment by johanandren** [comment]
Lets just close this since it's already in the README

**Pull request closed without merging**